### PR TITLE
feat(isemail): adding isEmail message

### LINKED ISSUE #29: add translation to IsEmail

### PR DESCRIPTION
**Pull request by bigotto**

### CODE REVIEW

File: src/messages/isEmail/isEmail.spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      email!: string;
+    }
+
+    const user = new User();
+    user.email = 'user.com';
+
+    const errors = validateSync(user);
+    const got = errors[0].constraints?.isEmail;
+
+    expect(got).toBe(want);
```
Comment (bigotto) [inline src/messages/isEmail/isEmail.spec.ts:31]: There is a problem here. @vini-barba can you look it?

Comment (bigotto) [reply src/messages/isEmail/isEmail.spec.ts:31]: Solved!!

**Comment by bigotto** [comment]
@vini-barba resolved conflict! Can u check it again?

File: src/config/locales/pt.json [hunk 2]
```diff
@@ -1,7 +1,8 @@
 {
   "Contains": "{{property}} deve conter a string '{{constraint}}'",
-  "IsMongoId": "{{property}} deve ser um mongodb id",
+  "IsMongoId": "O valor {{value}} não é um MongoID válido",
```
Comment (bigotto) [inline src/config/locales/pt.json]:
```suggestion
  "IsMongoId": "{{property}} deve ser um mongodb id",
```

File: src/messages/isEmail/isEmail.spec.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { IsEmail, validateSync } from 'class-validator';
+import isEmailMessage from './isEmail';
+
+describe('Contains', () => {
```
Comment (vini-barba) [inline src/messages/isEmail/isEmail.spec.ts]:
```suggestion
describe('IsEmail', () => {
```

File: src/config/locales/pt.json [hunk 4]
```diff
@@ -2,6 +2,7 @@
   "Contains": "{{property}} deve conter a string '{{constraint}}'",
   "IsMongoId": "{{property}} deve ser um mongodb id",
   "IsAlpha": "{{property}} deve conter apenas letras (a-zA-Z)",
+  "IsEmail": "{{property}} deve ser um email"
```
Comment (vini-barba) [inline src/config/locales/pt.json]:
```suggestion
  "IsEmail": "{{property}} deve ser um email",
```

**Review by vini-barba — CHANGES_REQUESTED** [review]

**Comment by vini-barba** [comment]
closes #29 #100

**Pull request merged**